app/components/settings/integration_component.rb

```diff
@@ -12,4 +12,8 @@ class Settings::IntegrationComponent < ApplicationComponent
   def sentinel_connected_class
     "#{title.downcase.gsub(' ', '-')}-integration-connected"
   end
+
+  def sync_in_progress?
```

Comment:
I think we need to scope by user here else this will fetch global latest in progress

Reply:
You are right!

Reply:
Done